trustpoint/users/signals.py

```diff
@@ -0,0 +1,32 @@
+import threading
+import os
+from django.core.signals import request_started
+from django.db.models.signals import post_migrate, post_init
+from django.dispatch import receiver
+from users.scheduler import TaskScheduler
+import logging
+
+logger = logging.getLogger('tp.users')
+
+tasks_initialized = False
+
+@receiver(request_started)
```

Comment:
What is the rationale for initializing the tasks on the first request rather than on server start? Even though this uses threading, it might still negatively affect the performance. More concerningly, if the trustpoint is restarted, but not accessed for a long time, periodic tasks will not run.

Reply:
To my understanding this is not supported by Django signals right now: https://docs.djangoproject.com/en/5.1/ref/signals/ 

Do you think `connection_created` would work?

Reply:
Yes, that works well! We also use it for CRL scheduling (which may be integrated in the task scheduler at a later time, but i would leave it separate for now)